README.md

```diff
@@ -1,16 +1,19 @@
 Developer guide to Tessel 2 Docs Gitbook
 ============
 
-# Tools
-Install the following command line tools using `npm i {{toolname}} -g`
```

Comment:
We still need install instructions. In order to use a Node command line tool, you have to install it globally